selenium_testing_library/screen.py

```diff
@@ -40,15 +42,23 @@ class NoSuchElementException(Exception):
 }
 
 
+class ElementsFinder(Protocol):
+    def find_elements(
+        self, by: str = locators.By.ID, value: str = None
+    ) -> List[WebElement]:
+        ...
+
+
 class Screen:
-    def __init__(self, driver: Driver):
+    def __init__(self, driver: RemoteWebDriver):
```

Comment:
Also I don't think it's correct that this is `RemoteWebDriver`. Should be some sort of a `T = TypeVar("T")` to make sure that when you pass in a ChromiumDriver (which extends RemoteWebDriver) that you have access to all the extra Chromium methods through screen.driver.